### PR TITLE
Sample siem fix

### PR DESCRIPTION
**Pull request by aviadl**

### CODE REVIEW

File: Integrations/integration-sampleSiem.yml [hunk 1]
```diff
@@ -86,7 +82,7 @@ script:
                     var incidentType = getIncidentType();
                     var incident = buildIncident(incidentType);
                     incident.name = 'Sample Incident - ' + incidentType;
-                    incident.rawJSON = JSON.stringify({incident});
```
Comment (asafshen) [inline Integrations/integration-sampleSiem.yml]: how did it worked before??

Comment (aviadl) [reply Integrations/integration-sampleSiem.yml]: Didnt

**Comment by dorsha** [comment]
@aviadl shouldn't be merged yet, the category is wrong:

image

File: Integrations/integration-sampleSiem.yml [hunk 2]
```diff
@@ -4,19 +4,15 @@ commonfields:
 name: Sample Incident Generator
 display: Sample Incident Generator
 category: SIEM & Analytics
```
Comment (dorsha) [inline Integrations/integration-sampleSiem.yml]: Change to Analytics & SIEM

**Pull request merged**